week-2/extended-project/valeria_and_friends/jsons/people.json

```diff
@@ -0,0 +1,246 @@
+      "height": "165",
+      "mass": "75",
+      "hair_color": "brown",
+      "skin_color": "light",
+      "eye_color": "blue",
+      "birth_year": "47BBY",
+      "gender": "female",
+      "homeworld": "https://swapi.co/api/planets/1/",
+      "films": [
+        "https://swapi.co/api/films/5/",
+        "https://swapi.co/api/films/6/",
+        "https://swapi.co/api/films/1/"
+      ],
+      "species": ["https://swapi.co/api/species/1/"],
+      "vehicles": [],
+      "starships": [],
+      "created": "2014-12-10T15:53:41.121000Z",
+      "edited": "2014-12-20T21:17:50.319000Z",
+      "url": "https://swapi.co/api/people/7/"
+    },
+    {
+      "name": "R5-D4",
+      "height": "97",
+      "mass": "32",
+      "hair_color": "n/a",
+      "skin_color": "white, red",
+      "eye_color": "red",
+      "birth_year": "unknown",
+      "gender": "n/a",
+      "homeworld": "https://swapi.co/api/planets/1/",
+      "films": ["https://swapi.co/api/films/1/"],
+      "species": ["https://swapi.co/api/species/2/"],
+      "vehicles": [],
+      "starships": [],
+      "created": "2014-12-10T15:57:50.959000Z",
+      "edited": "2014-12-20T21:17:50.321000Z",
+      "url": "https://swapi.co/api/people/8/"
+    },
+    {
+      "name": "Biggs Darklighter",
+      "height": "183",
+      "mass": "84",
+      "hair_color": "black",
+      "skin_color": "light",
+      "eye_color": "brown",
+      "birth_year": "24BBY",
+      "gender": "male",
+      "homeworld": "https://swapi.co/api/planets/1/",
+      "films": ["https://swapi.co/api/films/1/"],
+      "species": ["https://swapi.co/api/species/1/"],
+      "vehicles": [],
+      "starships": ["https://swapi.co/api/starships/12/"],
+      "created": "2014-12-10T15:59:50.509000Z",
+      "edited": "2014-12-20T21:17:50.323000Z",
+      "url": "https://swapi.co/api/people/9/"
+    },
+    {
+      "name": "Obi-Wan Kenobi",
+      "height": "182",
+      "mass": "77",
+      "hair_color": "auburn, white",
+      "skin_color": "fair",
+      "eye_color": "blue-gray",
+      "birth_year": "57BBY",
+      "gender": "male",
+      "homeworld": "https://swapi.co/api/planets/20/",
+      "films": [
+        "https://swapi.co/api/films/2/",
+        "https://swapi.co/api/films/5/",
+        "https://swapi.co/api/films/4/",
+        "https://swapi.co/api/films/6/",
+        "https://swapi.co/api/films/3/",
+        "https://swapi.co/api/films/1/"
+      ],
+      "species": ["https://swapi.co/api/species/1/"],
+      "vehicles": ["https://swapi.co/api/vehicles/38/"],
+      "starships": [
+        "https://swapi.co/api/starships/48/",
+        "https://swapi.co/api/starships/59/",
+        "https://swapi.co/api/starships/64/",
+        "https://swapi.co/api/starships/65/",
+        "https://swapi.co/api/starships/74/"
+      ],
+      "created": "2014-12-10T16:16:29.192000Z",
+      "edited": "2014-12-20T21:17:50.325000Z",
+      "url": "https://swapi.co/api/people/10/"
+    }
+  ]
+}
```

Comment:
It's a good idea to store the data of api in files but only if it is used for backup purpose, because the idea of ​​apis is use them int runtime of your project, It would be a good option if all this information was assigned in a variable in your project.

Reply:
you're right. we let it slip. thanks for making us notice it :)